src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -308,6 +317,52 @@ export class Tv2CasparCgTimelineObjectFactory implements Tv2GraphicsElementTimel
     }
   }
 
+  public createAudioBedTimelineObject(audioBedName: string, blueprintConfiguration: Tv2BlueprintConfiguration): CasparCgMediaTimelineObject {
+    const audioBedSettings: AudioBedSettings = blueprintConfiguration.studio.audioBedSettings
+    const audioBedConfiguration: AudioBedConfiguration | undefined = blueprintConfiguration.showStyle.audioBedConfigurations.find(audioBedConfiguration => audioBedConfiguration.name === audioBedName)
+    if (!audioBedConfiguration) {
+      throw new NotFoundException(`Unable to find the audio bed configuration for ${audioBedName}.`)
+    }
+    return {
+      id: 'casparCg_audio_bed',
+      enable: {
+        start: 0
+      },
+      layer: Tv2CasparCgLayer.AUDIO,
+      priority: 1,
+      content: {
+        deviceType: DeviceType.CASPAR_CG,
+        type: CasparCgType.MEDIA,
+        file: this.assetPathHelper.joinAssetToFolder(audioBedConfiguration.filename, audioBedSettings.mediaDirectory),
+        channelLayout: AUDIO_CHANNEL_LAYOUT,
+        loop: true,
+        noStarttime: true,
+        mixer: {
+          volume: audioBedSettings.volume / 100,
+        },
+        transitions: {
+          inTransition: {
+            type: CasparCgTransitionType.MIX,
+            easing: CasparCgTransitionEase.LINEAR,
+            direction: CasparCgTransitionDirection.LEFT,
+            duration: this.convertFramesToMilliseconds(audioBedConfiguration.fadeInDurationInFrames),
+          },
+          outTransition: {
+            type: CasparCgTransitionType.MIX,
+            easing: CasparCgTransitionEase.LINEAR,
+            direction: CasparCgTransitionDirection.LEFT,
+            duration: this.convertFramesToMilliseconds(audioBedConfiguration.fadeOutDurationInFrames),
+          },
+        }
+      },
+      classes: ['lyd_on_air'],
```

Comment:
A thing I have been trying to do while moving stuff from Blueprints is to see how many of these `classes` we can remove.
The sole purpose of `classes` on TimelineObjects is to help with figuring out when to put the TimelineObject on Air (just like `priority` and `enable`).
It could for example look like this:
```
cons timelineObjectA = {
  id: 'a',
  enable: {
    start: 0,
    end: 1000
  },
  classes: ['some_class'],
  content: ...
}

const timelineObjectB = {
  id: 'b',
  enable: {
    start: '.some_class.end'
  },
  content: ...
}
```
The above snippet makes `timelineObjectB` start when `timelineObjectA` ends.

Looking through old Blueprints it looks like that `lyd_on_air` is only to stop the baseline audio object while we are playing an audio bed. 

Would be nice if we could test if this `lyd_on_air` is needed :) 

Note: It's not that the concept of `classes` is bad. Looking through old Blueprints it just haven't been utilized in a good way. (We do use `classes` for some things in the new Blueprints)

Reply:
A very good point. I also had the thought of whether it is necessary or not. I have created a TODO on that line to check if it is necessary.